samples/attested_tls/common/cert_verifier.cpp

```diff
@@ -11,7 +11,19 @@
 #include <mbedtls/platform.h>
 #include <mbedtls/ssl.h>
 #include <string.h>
-#include "../../common/utility.h"
+#include "utility.h"
+
+#if defined(CLIENT_CERT_VERIFY_CALLBACK)
```

Comment:
Consider avoiding the use of #ifdefs for readability, particularly because this is in sample code. For example, add a `#include <tls_enclave_config.h>` where the tls_enclave_config.h file in the client and server folders (explicitly pathed as an include search path in the appropriate Makefile) include the appropriate headers and definitions instead. This specialization header file can then also be reused in identity_verifier.cpp.